api/api_test.go

```diff
@@ -0,0 +1,38 @@
+package api
```

Comment:
For tests, should we consider doing the `Arrange`, `Act`, `Validate` comments schema?

Reply:
yeah I will do that, didn't do that because I thought I am going to overwrite the current empty function really soon though. I will update it!